<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profiler</class>
 <widget class="QWidget" name="Profiler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTreeWidget" name="profilerTree">
     <column>
      <property name="text">
       <string>Funciton</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Max</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Min</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Average</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Calls</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toggleProfilerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/toolbar/icons/run.png</normaloff>:/toolbar/icons/run.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="bottomUpToggle">
       <property name="text">
        <string>Bottom Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="totalRadioButton">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="stepRadioButton">
       <property name="text">
        <string>Step</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
